Guess_My_Age/data_generation_age.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pandas as pd
+import random
```

Comment:
from /utils import file_compression

Reply:
I created a top-level script instead that handles all the compression. Is that ok?